dialogue-apache-hc5-client/src/main/java/com/palantir/dialogue/hc5/TlsProtocols.java

```diff
@@ -16,26 +16,14 @@
 
 package com.palantir.dialogue.hc5;
 
-import com.google.common.primitives.Ints;
-
 /** Internal utility functionality to slowly roll out new TLS protocol support. */
 final class TlsProtocols {
 
-    private static final boolean JAVA_15_OR_LATER = isJava15OrLater();
```

Comment:
Should we keep a check here that the client JDK is 11.0.3 or later? Probably not strictly necessary if we can provide some guarantees on the server side?

Reply:
I've gone back and forth on the client-side a few times, heh. On the one hand, I'd like to ensure we can safely deprecate tlsv1.2 asap (no plans to do so, but we often don't get a long runway). However, if folks do run on old jdks in odd environments, it's difficult to tell, and we don't actually expect to see any 11.0.2 or older in our managed environments, so the two _should_ be equivalent.

I might as well add the check here since we don't expect to see it, and if our expectations are incorrect, we won't see odd tls failures.